Packs/SilentPush/Integrations/SilentPush/SilentPush.py

```diff
@@ -4258,12 +4267,14 @@ def add_indicators_tags_command(client: Client, args: dict[str, Any]) -> Command
         CommandResults: JSON response of added indicator tags.
     """
     result = client.add_indicators_tags(args)
+    tags = args.get('tags')
+    indicator_name = args.get('indicator_name')
 
     return CommandResults(
         outputs_prefix="SilentPush.AddIndicatorTags",
         outputs_key_field="feed_uuid",
         outputs=result,
-        readable_output=tableToMarkdown("SilentPush Add Indicator Tags", result),
+        readable_output=tableToMarkdown(f"Indicator Tags: '{tags}' added to indicator '{indicator_name}' successfully", result),
```

Comment:
no need for the table:
readable_output=f"Indicator Tags: '{tags}' added to indicator '{indicator_name}' successfully"